pkg/sql/parser/sql.y

```diff
@@ -2569,6 +2571,15 @@ zone_value:
     $$.val = tree.NewStrVal($1)
   }
 
+inject_stats_stmt:
+  INJECT STATISTICS FOR TABLE table_name a_expr
```

Comment:
I wonder if this should be an `ALTER TABLE` variant. Something like `ALTER TABLE <table> INJECT STATISTICS a_expr`.

Reply:
I'm open to that. @knz what do you think?

Also, I'm not sure if I should make the `StatementType` be `Ack` or `DDL` (or what the consequence of choosing one over the other is). Also not entirely sure what privilege to require (currently INSERT; maybe it should be  CREATE like for alter table?).

Note that I plan to add support for this statement in the optimizer test catalog (`exec-ddl` test 
 directive), similar to the support for CREATE TABLE.